BUILDING.txt

```diff
@@ -6,6 +6,7 @@ Requirements:
 * Unix System
 * JDK 1.8
 * Maven 3.3 or later
+* Boost 1.76
```

Comment:
Would you add `(if compiling native code)`, as well as Protocol Buffers, CMake, and so on?